cmd/aitaskbuilder/get.go

```diff
@@ -0,0 +1,95 @@
+package aitaskbuilder
+
+import (
+	"errors"
+	"fmt"
+	"io"
+
+	"github.com/prolific-oss/cli/client"
+	"github.com/spf13/cobra"
+)
+
+// BatchGetOptions is the options for the get aitaskbuilder batch command.
+type BatchGetOptions struct {
+	Args           []string
+	BatchID        string
+	NonInteractive bool
+}
+
+// NewGetCommand creates a new `aitaskbuilder get` command to get details about
+// a specific AI task builder batch.
+func NewGetCommand(client client.API, w io.Writer) *cobra.Command {
+	var opts BatchGetOptions
+
+	cmd := &cobra.Command{
+		Use:   "get",
+		Short: "Get an AI task builder batch",
+		Long: `Get details about a specific AI task builder batch
+
+This command allows you to retrieve details of a specific AI task builder batch by providing
+the batch ID.`,
+		Example: `
+Get an AI task builder batch:
+$ prolific aitaskbuilder get -b <batch_id>
+		`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			opts.Args = args
+
+			err := renderAITaskBuilderBatch(client, opts, w)
+			if err != nil {
+				return fmt.Errorf("error: %s", err.Error())
+			}
+
+			return nil
+		},
+	}
+
+	flags := cmd.Flags()
+	flags.StringVarP(&opts.BatchID, "batch-id", "b", "", "Batch ID (required) - The ID of the batch to retrieve.")
+	flags.BoolVarP(&opts.NonInteractive, "non-interactive", "i", false, "Render the batch details straight to the terminal.")
```

Comment:
I don't think this is used yet. You thinking of using charm to render an interactive model at some point?